pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -90,6 +90,9 @@ static int _format(vfs_mount_t *mountp)
 
     /* make sure the volume has been initialized */
     if (_init(mountp)) {
+        #ifndef CONFIG_FATFS_FORMAT_ALLOC_STATIC
+        free(work);  
+        #endif
```

Comment:
CI doesn't like the trailing whitespace.
You can also use the `IS_USED()` macro to make this a C-conditional instead which looks a bit nicer and has the advantage that the compiler will check both branches (although that's not relevant here). 

```suggestion
        if (!IS_USED(CONFIG_FATFS_FORMAT_ALLOC_STATIC)) {
            free(work);
        }
```